radixdlt/src/main/java/org/radix/GlobalInjector.java

```diff
@@ -192,7 +201,35 @@ public LocalSystemProvider(SystemCounters counters, RuntimeProperties properties
 		public LocalSystem get() {
 			String host = this.hostIp.hostIp()
 				.orElseThrow(() -> new IllegalStateException("Unable to determine host IP"));
-			return LocalSystem.create(this.counters, this.properties, this.universe, host);
+			Supplier<Map<String, Object>> infoSupplier = () -> {
```

Comment:
This really needs to go into a separate class, or at the very least a separate method.